selenium/tests/test_selenium.py

```diff
@@ -4,7 +4,9 @@
 from testcontainers.core.utils import is_arm
 
 
-@pytest.mark.parametrize("caps", [DesiredCapabilities.CHROME, DesiredCapabilities.FIREFOX])
+@pytest.mark.parametrize("caps",
+                         [DesiredCapabilities.CHROME, DesiredCapabilities.FIREFOX],
+                         ids=['chrome', 'ff'])
```

Comment:
I think this highlights, that above was a breaking API change. Can we do anything about this?

Reply:
all im doing here is naming the test cases, so you can see which one fails, if one of them does fail, purely cosmetic